docs/index.html

```diff
@@ -1814,13 +1814,7 @@ <h2 id="triggerssearchescreates">Triggers/Searches/Creates</h2>
     <div class="col-md-5 col-sm-12 col-height  docs-primary">
       <p>You can find more details on the definition for each by looking at the <a href="https://github.com/zapier/zapier-platform/blob/master/packages/schema/docs/build/schema.md#triggerschema">Trigger Schema</a>,
 <a href="https://github.com/zapier/zapier-platform/blob/master/packages/schema/docs/build/schema.md#searchschema">Search Schema</a>, and <a href="https://github.com/zapier/zapier-platform/blob/master/packages/schema/docs/build/schema.md#createschema">Create Schema</a>.</p><blockquote>
-<p>Example App: Check out <a href="https://github.com/zapier/zapier-platform/tree/master/example-apps/trigger">https://github.com/zapier/zapier-platform/tree/master/example-apps/trigger</a> for a working example app using triggers.</p>
-</blockquote><blockquote>
-<p>Example App: Check out <a href="https://github.com/zapier/zapier-platform/tree/master/example-apps/rest-hooks">https://github.com/zapier/zapier-platform/tree/master/example-apps/rest-hooks</a> for a working example app using REST hook triggers.</p>
-</blockquote><blockquote>
-<p>Example App: Check out <a href="https://github.com/zapier/zapier-platform/tree/master/example-apps/search">https://github.com/zapier/zapier-platform/tree/master/example-apps/search</a> for a working example app using searches.</p>
-</blockquote><blockquote>
-<p>Example App: Check out <a href="https://github.com/zapier/zapier-platform/tree/master/example-apps/create">https://github.com/zapier/zapier-platform/tree/master/example-apps/create</a> for a working example app using creates.</p>
+<p>To create a new integration with a premade trigger, search, or create, run <code>zapier init [your app name]</code> and select from the list that appears. You can also check out our working example apps <a href="https://github.com/zapier/zapier-platform/tree/master/example-apps">here</a>.</p>
```

Comment:
@tylerrobertsonzapier Should we also direct devs to use `zapier scaffold` after their `init` as mentioned in [this comment](https://github.com/zapier/zapier-platform/pull/530#pullrequestreview-940056946)?

Reply:
@reganstarr Great call out! I don't think `zapier scaffold` is strictly necessary here, as the templates with `init` will already have scaffolded triggers/actions/auths/etc. added in. Unless I'm wrong about that?

Regardless: we should definitely make sure we're calling out `zapier scaffold` _somewhere_ as a way to quickly add triggers and actions.

Reply:
Not quite, the minimal example is _truly_ minimal. This is the `index.js`:

```js
module.exports = {
  // This is just shorthand to reference the installed dependencies you have.
  // Zapier will need to know these before we can upload.
  version: require('./package.json').version,
  platformVersion: require('zapier-platform-core').version,

  // If you want your trigger to show up, you better include it here!
  triggers: {},

  // If you want your searches to show up, you better include it here!
  searches: {},

  // If you want your creates to show up, you better include it here!
  creates: {},

  resources: {},
};
```

So scaffolding should definitely be part of the story!

Reply:
I tried out `zapier init` for custom-auth and it did not create any triggers/actions/searches.

Reply:
Thank you both! Updating that line that @reganstarr pointed out to include `zapier scaffold`.